src/util/resourceLimits.tsx

```diff
@@ -0,0 +1,74 @@
+import ResourceLink from "components/ResourceLink";
+
+export type ResourceLimitSource = "project" | "cluster-member";
+export interface ResourceLimit {
+  min: number;
+  max: number;
+  source?: ResourceLimitSource;
+}
+
+export const getResourceLimit = (
+  resources: number[],
+  isClusterMemberDefined: boolean,
```

Comment:
```suggestion
  clusterMember: string | undefined,
```
I think passing in the member and checking on it below is bit simpler.